source/frontend/src/features/leases/models.ts

```diff
@@ -160,74 +166,43 @@ export class LeaseDetails {
 
 export class FormLeaseProperty {
   id?: number;
-  property?: FormProperty;
+  property?: PropertyForm;
   leaseId?: number;
   rowVersion?: number;
-  landArea?: number;
-  areaUnitType?: Api_TypeCode<string>;
+  landArea: string;
+  areaUnitTypeCode: string;
 
-  public constructor(leaseId?: number) {
-    this.property = new FormProperty();
+  private constructor(leaseId?: number) {
     this.leaseId = leaseId;
+    this.landArea = '0';
+    this.areaUnitTypeCode = PropertyAreaUnitTypes.Meter;
   }
 
   static fromApi(apiPropertyLease: Api_PropertyLease): FormLeaseProperty {
     const model = new FormLeaseProperty(apiPropertyLease.lease?.id);
-    model.property = FormProperty.fromApi(apiPropertyLease.property ?? {});
+    model.property = PropertyForm.fromApi(apiPropertyLease.property ?? {});
     model.id = apiPropertyLease.id;
     model.rowVersion = apiPropertyLease.rowVersion;
-    model.landArea = apiPropertyLease.leaseArea;
-    model.areaUnitType = apiPropertyLease.areaUnitType;
+    model.landArea = apiPropertyLease.leaseArea?.toString() || '0';
```

Comment:
hmm, I slightly worry that a 0 is not equal to not set in this case. We had a similar problem in CITZ pims, where setting a value of zero had significance vs. not setting at all.

Reply:
kk